src/inmanta/moduletool.py

```diff
@@ -159,7 +158,7 @@ def get_project(self, load: bool = False, strict_deps_check: Optional[bool] = No
             project.load()
         return project
 
-    def determine_new_version(self, old_version, version, major, minor, patch, dev):
+    def determine_new_version(self, old_version, version, major, minor, patch, revision, dev):
```

Comment:
Yes, we don't want this change for this method. I made them because this method also used the `parse_from_bools` method which required the revision boolean.